src/Timeline.tsx

```diff
@@ -123,6 +133,12 @@ export const Timeline = <EID extends string, LID extends string>({
 
           const timeAtCursor = timeScale.invert(mousePosition.x)
 
+          useEffect(() => {
+            if (onZoomRangeMove) {
+              onZoomRangeMove(timeAtCursor - zoomWidth / 2, timeAtCursor + zoomWidth / 2)
```

Comment:
* Use same logic as lines 148/149 in order not extend outside of domain
* Properly cover rubber-band mode (Shift + Click + Drag) in which cursor does not show a fixed zoom band